ui/app/AppLayouts/Profile/stores/DevicesStore.qml

```diff
@@ -11,6 +11,10 @@ QtObject {
     // Module Properties
     property bool isDeviceSetup: devicesModule.isDeviceSetup
 
+    function onSectionSetActive(name) {
+        return root.devicesModule.onSectionSetActive()
+    }
```

Comment:
Shouldn't it be driven by the backend? We usually propagate that information from main module.

Reply:
+ where's the `name` param?

Reply:
Good point.
I tried doing the loading from the `view`, when we try to access the model, but it seems like we do from the app start, which is not good.
We currently don't have the architecture where the profile sections are switched from the backend and I feel like it's a too big endeavor for this particular task.
IMO, we should do a refactor of the profile sections later in a more thorough way, because just this small section needs a lot of love. I tried to improve what I could (using a column instead of anchors, fixing warnings), but it's needs more work.